### ISSUE TITLE
2.4.11: Confusing wording due to unconventional state requirement

### ISSUE THREAD

**Issue by benja11y** [w3c/wcag#1859]
The logical structure of the SC is hard to follow, specifically because interpreting it correctly appears to require considering the page background as part of the control—something that we don't think will feel natural for most people. If doing so is required, the normative wording should clearly signpost this.

Two relevant parts from the SC wording:

1. The content under test must meet all four of the requirements (including "**Contrasting area:** There is an area of the focus indicator that has a contrast ratio of at least 3:1 between the colors in the focused and unfocused states.")
2. The "**Adjacent contrast**" clause has two parts, joined by an "or", so you can pick either one.

The choice in the "Adjacent contrast" requirement is subtle and risks being misinterpreted—perhaps making the two options under "Adjacent contrast" into bullet points (as has been done with "Minimum area") would make it clearer?

Also, Figure 15 in the ["Adjacent contrast" section of the Understanding doc](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum#Contrast-thickness) initially appears to violate the first requirement, as the 2px border added to the control to act as the focus indicator doesn't contrast with the rest of the button.

However, the explanation for Figure 18 in the ["Relationship to Non-text Contrast" section of the Understanding doc](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum#relation-non-text-color-contrast), which features an indication that doesn't contrast 3:1 with the border of the checkbox, says:

> The pixels making up the blue focus indicator have a sufficient Change of contrast from their white color in the unfocused state 

Thus it seems the intent of the SC is for us to consider the white (in this case) page background as part of the checkbox's unfocused state. Doing so allows all of the above to fit together.

However, the fact that the page background is considered part of the control feels counter-intuitive and is not explicitly called out in the normative text. Other SCs use the phrase "adjacent colors" to help clarify this, though explicitly mentioning "including the page background" would be clearer.

This issue has been raised on behalf of [TPGi](https://www.tpgi.com), based on an internal consultation process. [An article containing all of our comments](https://www.tpgi.com/wcag-22-working-draft-feedback/) can be found on our blog, along with an introduction and acknowledgments.

**Comment by alastc** [comment]
> appears to require considering the page background as part of the control

The "Contrasting area" requirement applies whether the indicator goes over the background, or not. 

I don't think it would help to talk about the background, because that may not be relevant. Some focus indicators appear within the component, and it has to apply in that case as well.

> Figure 15 in the "Adjacent contrast" section of the Understanding doc initially appears to violate the first requirement, as the 2px border added to the control to act as the focus indicator doesn't contrast with the rest of the button.

It meets "Contrasting area" (previously called "change of contrast") because the pixels making up the added black outline are changed from white (the background). It is there to demonstrate that you don't have to also have adjacent-contrast with the component, if it is thick enough.

> the fact that the page background is considered part of the control feels counter-intuitive and is not explicitly called out in the normative text. Other SCs use the phrase "adjacent colors" to help clarify this, though explicitly mentioning "including the page background" would be clearer.

It is not possible to use 'adjacent' in the first bullet because the key factor is the _change_. Focus indicators are dynamic and the main reason this SC is needed is because using adjacent is not sufficient. (See this [test page](https://alastairc.uk/tests/wcag22-examples/focus-more-visible.html) which looked at some basic examples to see if using adjacent colours could work.)

**Comment by DavidMacDonald** [comment]
Agree with @alastc response. 

However we must acknowledge at least to ourselves that this is likely the most confusing SC of all of WCAG. Necessarily so if we want to strengthen the requirements for focus indicators without hamstringing authors, but nonetheless confusing, even for a11y  experts. I'm not sure what can be done about that.

@benja11y 
Given this response by @alastc , do you have a suggestion on what to add or remove from the SC that would help with understanding the SC without changing the requirements.

**Comment by alastc** [comment]
Thanks @DavidMacDonald,

I think (similar to some other comments) that putting some passing examples right at the top of the understanding is probably the best.

**Comment by alastc** [comment]
I've made an update to the top of the understanding in #2000.

**Comment by alastc** [comment]
The group agreed with the response(s).
